packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -212,6 +209,49 @@ export function BlockSettingsDropdown( {
 					className="block-editor-block-settings-menu"
 					popoverProps={ POPOVER_PROPS }
 					noIcons
+					menuProps={ {
+						/**
+						 * @param {KeyboardEvent} event
+						 */
+						onKeyDown( event ) {
```

Comment:
Is the only reason for this callback to keep the focus in the list view after deleting/duplicating/inserting a block? Or is there another reason?

Reply:
It's to allow pressing keyboard shortcuts when the focus is in the menu dropdown. I couldn't find any other way to do this without having to duplicate a lot of the code 😢 .